web/app/components/post-item.js

```diff
@@ -708,7 +708,7 @@ function findVimeoID(url) {
  * @returns {string|null}
  */
 function findTweetID(url) {
-    if (url.hostname !== "twitter.com") {
+    if (!["twitter.com", "x.com", "mobile.twitter.com"].includes(url.hostname)) {
```

Comment:
just FYI; "mobile.twitter.com" is redundant as it's already matched by "twitter.com"

Reply:
Ah, checked but it's not redundant. `hostname` in this case can return `mobile.twitter.com`.

Reply:
ah, my bad
I got things mixed up
it would work if it was the other way around (`url.hostname.includes("twitter.com")`)
but you're right, it's not redundant for the current code

Reply:
you could do something like
`!["twitter.com", "x.com"].find((d) => url.hostname.includes(d))`
this would also include the mobile subdomain, both for the twitter and the x domains

Reply:
Almost there. That code would also match `nottwitter.com` and `notx.com` :p

Reply:
At this point I don't know if `mobile.` is a thing anymore. The issue was opened a while ago. Now it looks like it redirects to the top domain...